fanficfare/adapters/adapter_fictionlive.py

```diff
@@ -116,6 +114,21 @@ def extract_metadata(self, data, get_cover):
 
         # nearly everything optional from here out
 
+        if 'storyStatus' in data:
+            self.story.setMetadata('status', data['storyStatus'])
+        elif 'complete' in data:
+            if data['complete'] == True:
+                self.story.setMetadata('status', "finished")
```

Comment:
Please use `Completed` and `In-Progress` for `status`.

While `status` can be set to other values as well, FFF assumes that `Completed` and `In-Progress` are standard and there's code in the plugin for setting boolean columns that expects them.

Reply:
Ok, will do. So should I always translate what the site gives, then? If so, what's the equivilent of `hiatus`?

Reply:
`Completed` and `In-Progress` are the only 'standard' values.  Personally, I'd capitalize any other values (Python's `title()` should be sufficient), but I won't insist on it.